website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -0,0 +1,683 @@
+    imageAlt="System Context"
+    zoomInPic={useBaseUrl('img/zoom-in.svg')}
+    zoomOutPic={useBaseUrl('img/zoom-out.svg')}
+    resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
+/>
+
+
+## Possible Technologies for Enabling the Work
+
+The following is a short list of technologies that could be leveraged during this
+work. These technologies fall within current architectures for Cumulus and
+extensions and may contribute to more rapid development of features as we progress.
+A full list of AWS services available in NGAP is available [here](https://wiki.earthdata.nasa.gov/display/ESKB/AWS+Services+Approval+Status+Page).
+
+
+### API Gateway
+
+The ORCA API Gateway provides an entry point for the Cumulus dashboard and users
+to interact directly with ORCA. For more information on the ORCA API gateway see
+the [API Gateway research page](research-APIGateway.md). All reconciliation
+calls from Cumulus should occur through the ORCA API Gateway.
+
+
+### GraphQL
+
+[GraphQL](https://graphql.org/learn/) is a query language for APIs. This is an
+additional research spike topic. The promise of this route further consolidates
+our database read/write code while at the same time providing a single API endpoint
+for clients (both ORCA and Cumulus) to retrieve the data from ORCA. This technology
+though adding some complexity could simplify some code and design aspects.
+
+Research aspects would include GraphQL server to use (COTS or home built). Impacts
+to AWS account (cost, speed, capabilities, security). Synergy with other projects
+like Cumulus.
+
+Some initial resources to start would include the following:
+
+- [GraphQL Servers (prebuilt)](https://www.moesif.com/blog/graphql/technical/Ways-To-Add-GraphQL-To-Your-Postgres-Database-Comparing-Hasura-Prisma-and-Others/)
+- [Build your Own GraphQL Server](https://www.howtographql.com/choose/)
+- [GraphQL classes](https://www.edx.org/course/exploring-graphql-a-query-language-for-apis)
+- [GraphQL Example with CMR](https://git.earthdata.nasa.gov/projects/CMRQL/repos/hackfest-cmr-graphql/browse)
+- [Using Apollo Server as a Lambda](https://www.apollographql.com/docs/apollo-server/deployment/lambda/)
+
+
+### SQS
+
+[AWS SQS FIFO queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html)
+should be used to manage the records to be written to the ORCA database. This
+allows the team to decouple the database code from several lambdas and centralize
+the code to one area. This may or may not be needed if GraphQL is utilized.
+
+
+### AWS PostgreSQL RDS Offerings
+
+To better align with Cumulus, ORCA will utilize the AWS PostgreSQL RDS offerings.
+See notes on the ORCA transition to the ORCA database [here](https://wiki.earthdata.nasa.gov/display/ORCA/ORCA+Migration+to+Cumulus+RDS).
+The resultant code from the reconciliation work should be able to handle working
+with a PostgreSQL backed. Both PostgreSQL RDS and Amazon Aurora are viable back end
```

Comment:
with a PostgreSQL backed. Both PostgreSQL RDS and Amazon Aurora are viable back end

Should "backed" be "back end"?

Reply:
yes it should be back end. fixed.